notes.tex

```diff
@@ -172,7 +172,7 @@ \section{Differential}
 
 Like integrals, derivatives are distributive over addition. There are some interesting rules that make working with derivatives easier: the product rule, the quotient rule, and the chain rule. 
 
-Let's start with the chain rule. If we have a function within a function in a derivative, like $\sin(\cos(x))$, we can assign a name to each function, like $g(x) = \cos(x)$ and $h(x) = \sin(x)$, and then follow this simple rule: $h'(g(x)) \cdot g'(x)$. So what does this mean? Well, let's plug it in. Remember that the $'$ symbol is pronounced prime, and is another way to write that we are taking the derivative. $\sin'(\cos(x)) \cdot \cos'(x)$. So the derivative of $\sin$ is $\cos$, so we write $\cos(\cos(x))$ for the first part, and the derivative of $\cos$ is $-\sin$, so we write $-\sin(x)$ for the second part. So now we have $\cos(\cos(x)) \cdot -\sin(x)$ which can be further simplified to $\cos^2(x) \cdot -\sin(x)$. This is the derivative of our original function.
+Let's start with the chain rule. If we have a function within a function in a derivative, like $\sin(\cos(x))$, we can assign a name to each function, like $g(x) = \cos(x)$ and $h(x) = \sin(x)$, and then follow this simple rule: $h'(g(x)) \cdot g'(x)$. So what does this mean? Well, let's plug it in. Remember that the $'$ symbol is pronounced prime, and is another way to write that we are taking the derivative. $\sin'(\cos(x)) \cdot \cos'(x)$. So the derivative of $\sin$ is $\cos$, so we write $\cos(\cos(x))$ for the first part, and the derivative of $\cos$ is $-\sin$, so we write $-\sin(x)$ for the second part. So now we have $\cos(\cos(x)) \cdot -\sin(x)$. This is the derivative of our original function.
```

Comment:
This is an example of why I'd recommend breaking to a new line after each sentence. It's impossible to see what actually changed here because the line that was changed is so long.